From 9e9fe812d2b1c905dbdedd7afe82ac12cd60a000 Mon Sep 17 00:00:00 2001
From: Zach Bacon <zachbacon@vba-m.com>
Date: Wed, 20 Dec 2023 17:11:19 -0500
Subject: [PATCH] visualboyadvance-m.metainfo.xml: add screenshots to the
 metainfo

Useful for app stores like KDE's discover and gnome software
Also updated to current specifications of appstream
Signed-off-by: Zach Bacon <zachbacon@vba-m.com>
---
 src/wx/visualboyadvance-m.metainfo.xml | 21 +++++++++++++++++----
 1 file changed, 17 insertions(+), 4 deletions(-)

diff --git a/src/wx/visualboyadvance-m.metainfo.xml b/src/wx/visualboyadvance-m.metainfo.xml
index 0e5cd4598..87c43f251 100644
--- a/src/wx/visualboyadvance-m.metainfo.xml
+++ b/src/wx/visualboyadvance-m.metainfo.xml
@@ -1,8 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component type="desktop">
- <id>visualboyadvance-m.desktop</id>
+<application>
+ <id type="desktop">visualboyadvance-m.desktop</id>
+ <developer id="vba-m.com">
+   <name>VisualBoyAdvance-M</name>
+ </developer>
  <metadata_license>CC-BY-SA-3.0</metadata_license>
  <project_license>GPL-2.0</project_license>
+ <content_rating type="oars-1.1"/>
  <name>VisualBoyAdvance-M</name>
  <summary>VisualBoyAdvance-M, a high compatibility Gameboy Advance Emulator</summary>
  <description>
@@ -15,7 +19,16 @@
     forks.
   </p>
  </description>
+ <screenshots>
+  <screenshot type="default">https://vba-m.com/screenshots/1.png</screenshot>
+  <screenshot>https://vba-m.com/screenshots/2.png</screenshot>
+  <screenshot>https://vba-m.com/screenshots/3.png</screenshot>
+  <screenshot>https://vba-m.com/screenshots/4.png</screenshot>
+ </screenshots>
+ <releases>
+   <release version="2.1.8" date="2023-12-14"/>
+ </releases>
  <url type="homepage">http://vba-m.com/</url>
  <url type="bugtracker">https://github.com/visualboyadvance-m/visualboyadvance-m/issues</url>
- <update_contact>zachbacon_at_vba-m.com</update_contact>
-</component>
+ <updatecontact>zachbacon@vba-m.com</updatecontact>
+</application>
